backend/inspirehep/records/serializers/jinja.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+import os
+from pathlib import Path
+
+import jinja2
+
+templates_path = Path(__file__).parent / "templates"
+
+jinja_cv_env = jinja2.Environment(
+    loader=jinja2.FileSystemLoader(os.path.abspath(templates_path)),
```

Comment:
sorry if I wasn't clear, I meant replacing all `os.path` by `pathlib` equivalents: https://docs.python.org/3/library/pathlib.html#correspondence-to-tools-in-the-os-module, so also this.